core/frontend/src/components/app/NetworkInterfacePriorityMenu.vue

```diff
@@ -3,7 +3,7 @@
     <v-list>
       <v-card-subtitle class="text-md-center" max-width="30">
         Move network interfaces over
-        to change network access priority
+        to change network access priority.<br>Applied changes require a <b>system reboot</b>.
```

Comment:
can we make it show the reboo-required icon we use for parameter changes? even in bold this is quite subtle.
a popup after saving saying "configurations will be applied on the next boot" should work well, too.

Reply:
> can we make it show the reboo-required icon we use for parameter changes? even in bold this is quite subtle. Or maybe a popup after saving saying "configurations will be applied on the next boot", if that is easier.

Yes, but that idea is under https://github.com/bluerobotics/BlueOS/pull/1962, to be a global notifier.